cypress/integration/rich-text/__snapshots__/RichTextEditor.Pasting.spec.ts.snap

```diff
@@ -2124,7 +2124,7 @@ exports[`Rich Text Editor > Microsoft Word (.docx) deserialization > paragraphs,
           "data": {},
           "marks": [],
           "nodeType": "text",
-          "value": "marks and"
+          "value": " marks and "
```

Comment:
is there any concern with the extra spaces here?
Maybe @z0al could have a look at this PR before merging?

Reply:
@chrishelgert could you explain why we needed to update the snapshot in this case?

Reply:
The whitespaces were removed from the previous replace regex, with the adjustment and only cleaning up tables this snapshot changed.
In this case it removed the space between <bold>, <a>, .. and regular text, which we wanted to fix